src/librustdoc/html/static/main.js

```diff
@@ -1424,6 +1424,17 @@ function getSearchElement() {
             return tmp;
         }
 
+        function colorCode(s) {
+            var parts = s.split("`");
```

Comment:
This seems _extremely_ fragile. We should be using the builtin markdown renderer for this